internal/apis/kfd/v1alpha2/eks/delete/infrastructure.go

```diff
@@ -49,19 +48,22 @@ func NewInfrastructure(dryRun bool, workDir, binPath string, kfdManifest config.
 }
 
 func (i *Infrastructure) Exec() error {
-	logrus.Info("Deleting infrastructure phase")
+	logrus.Info("Deleting infrastructure phase...")
 
 	err := iox.CheckDirIsEmpty(i.OperationPhase.Path)
 	if err == nil {
-		logrus.Infof("infrastructure phase already executed, skipping")
+		logrus.Infof("infrastructure phase already executed, skipping...")
 
 		return nil
 	}
 
 	err = i.tfRunner.Destroy()
 	if err != nil {
-		return fmt.Errorf("error running terraform destroy: %w", err)
+		return fmt.Errorf("error while deleting infrastructure: %w", err)
 	}
 
+	logrus.Warnf("Please, remember to kill the OpenVPN process if" +
+		" you have chosen to create it in the infrastructure phase")
```

Comment:
can we add a command here so the user can copy & paste it?

Reply:
Yep, do you think it's better to do that when we work on https://github.com/sighupio/furyctl/issues/140 ?
to be sure about needing sudo or not when killing the process

Reply:
makes sense 👍